src/_locales/de/messages.json

```diff
@@ -170,11 +170,11 @@
         "description": "Tooltip shown over the rightmost part of a domain slider."
     },
     "breakage_warning_tooltip": {
-        "message": "Das Blocken dieser Domain ist dafür bekannt, bei Websites Probleme zu verursachen",
+        "message": "Das Blockieren dieser Domain ist dafür bekannt, bei Websites Probleme zu verursachen",
```

Comment:
Is this a stylistic preference, or a grammar fix, or something else? I would like to avoid making changes unless they fix a problem.

Reply:
Blockieren would be the more official word, but still both are correct and can be used as synonym. I will just undo the change